### ISSUE TITLE
batch normalization+ReLu

### ISSUE THREAD

**Issue by yossitsadok** [fabianbormann/Tensorflow_Segmentation#17]
I have two questions regarding model implementation:

1. Is there any specific reason why batch norm is not implemented?
2. why deconv layers doesn't end with ReLu activation?

Thanks!

**Comment by fabianbormann** [comment]
closed by e4d59e9
